### DISCUSSION TITLE
JBD BMS will not be recognized

### DISCUSSION THREAD

**Discussion by sloesp** [Louisvdw/dbus-serialbattery#607] [Q&A] [answered]
Hello,
i am using a Jaibaida AP21S002 21S 300A BMS with the RS485/USB Interface. If i use the JBDTools on Windows i get Data from the BMS. With Venus OS on RASP-Pi i see the LEDs on the Interface blinking but the are not BMS Information at the GUI. 
The Logfile says:
@400000006453a4fd23a7a5bc INFO:SerialBattery:Testing LltJbd
@400000006453a4fd2a21742c INFO:SerialBattery:Testing Ant
@400000006453a4fd3a038a24 ERROR:SerialBattery:>>> ERROR: No reply - returning
@400000006453a4fd3a1600b4 ERROR:SerialBattery:>>> ERROR: Incorrect Data
@400000006453a4fd3a1f99bc INFO:SerialBattery:Testing Daly
@400000006453a4fe0e6c9e8c ERROR:SerialBattery:>>> ERROR: No reply - returning
@400000006453a4fe0e7ce684 INFO:SerialBattery:Testing Daly
@400000006453a4fe1e60bdb4 ERROR:SerialBattery:>>> ERROR: No reply - returning
@400000006453a4fe1e71154c INFO:SerialBattery:Testing Jkbms
@400000006453a4fe2e63b1bc ERROR:SerialBattery:>>> ERROR: No reply - returning
@400000006453a4fe2e741124 INFO:SerialBattery:Testing Lifepower
@400000006453a4ff02c8074c ERROR:SerialBattery:>>> ERROR: No reply - returning
@400000006453a4ff02d7c2a4 ERROR:SerialBattery:>>> ERROR: Incorrect Data
@400000006453a4ff02e17aec INFO:SerialBattery:Testing Renogy
@400000006453a4ff12c2f4a4 ERROR:SerialBattery:>>> ERROR: No reply - returning
@400000006453a4ff12d3a22c INFO:SerialBattery:Testing Renogy
@400000006453a4ff22b22de4 ERROR:SerialBattery:>>> ERROR: No reply - returning
@400000006453a4ff22c2a8a4 INFO:SerialBattery:Testing Ecs
@400000006453a500085e150c ERROR:SerialBattery:ERROR >>> No battery connection at /dev/ttyACM0

Any Ideas?
Thanks so far.
Slotty

**Comment by mr-manuel** [comment, marked as answer]
Please open a new issue for that. Go to Issues -> New Issue -> Support request.
